src/Compilers/CSharp/Portable/Symbols/DynamicTypeSymbol.cs

```diff
@@ -190,9 +193,9 @@ public override int GetHashCode()
             return (int)Microsoft.CodeAnalysis.SpecialType.System_Object;
         }
 
-        internal override bool Equals(TypeSymbol t2, TypeCompareKind comparison, IReadOnlyDictionary<TypeParameterSymbol, bool> isValueTypeOverrideOpt = null)
+        internal override bool Equals(TypeSymbol? t2, TypeCompareKind comparison, IReadOnlyDictionary<TypeParameterSymbol, bool>? isValueTypeOverrideOpt = null)
         {
-            if ((object)t2 == null)
+            if ((object?)t2 == null)
```

Comment:
>(object?)t2 == null [](start = 16, length = 19)

nit: prefer `is` check

Reply:
I tried to minimize extraneous changes in all of these PRs.